upcase_booklist_app/templates/lists.html

```diff
@@ -0,0 +1,28 @@
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+       <nav class="page-header">
+            <div class="header-items"><a href="{{ url_for('index') }}">Home</a></div> 
+           <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
+           <div class="header-items"><a href="{{ url_for('authors') }}">Authors</a></div>
+           <div class="header-items"><a href="{{ url_for('lists') }}">Lists</a></div>
+           
+       </nav>
+
+       <div>
+            {% for category in categories %}
+            <div class="category-title"<a href="{{ url_for('category', category_id=category['category_id'], category_title=category['category']) }}"><h1>{{ category['category'] }}</h1></a></div>
+                {% for item in category['books'] %}                    
+                    <a href="{{ url_for('book', book_id=item['id']) }}"><img src="{{ item['image'] }}"></a>
+            
+            {% endfor %}
```

Comment:
These should have different indentations that match their corresponding `for` loop tags.